specs/toolsets/adr_base/setup-toolset.sh

```diff
@@ -16,23 +16,8 @@ if [ -z "$ADR_VERSION"  ]; then
     return 1
 fi
 
-case $ADR_VERSION in
-    50)
-        export ADR_SDK_PLATFORM=android-21
-        export ADR_NDK_PLATFORM=android-21
-        ;;
-    42)
-        export ADR_SDK_PLATFORM=android-17
-        export ADR_NDK_PLATFORM=android-17
-        ;;
-    22)
-        export ADR_SDK_PLATFORM=android-8
-        export ADR_NDK_PLATFORM=android-8
-        ;;
-    *)
-        echo "E/Android toolset: Unsupported version: '${ADR_VERSION}' !"
-        return 1;
-esac
+export ADR_SDK_PLATFORM=android-$ADR_API
+export ADR_NDK_VERSION=23.1.7779620
```

Comment:
This looks scary and very specific but `sdkmanager --install "ndk;${ADR_NDK_VERSION}" --channel=3` should handle this correctly and always install the correct version.